generated/include/vnx/Endpoint.hxx

```diff
@@ -4,13 +4,19 @@
 #ifndef INCLUDE_vnx_Endpoint_HXX_
 #define INCLUDE_vnx_Endpoint_HXX_
 
+#ifdef _MSC_VER
+#include <vnx_base_export.h>
```

Comment:
where's this file?

Reply:
It is autogenerated with GENERATE_EXPORT_HEADER Cmake macro